### DISCUSSION TITLE
Is it possible to automatically alert the administrator when the 'Minimum Free Storage Space' settings are reached?

### DISCUSSION THREAD

**Discussion by genius9450** [datalust/seq-tickets#2167] [Help] [answered]
In the settings of SYSTEM, there is a configuration for "Minimum free storage space", which represents "To prevent system issues Seq will stop accepting events if free disk space falls below this threshold.".

So, when the system's free storage space reaches the configured value, how can we be notified of this situation? 
Can Seq proactively send notifications, such as via email or other channels?

Alternatively, could Seq clear old log data when the space is insufficient, allowing new logs to continue being written?

**Comment by nblumhardt** [comment, marked as answer]
Hi! There isn't any functionality along these lines available today; using an external check such as https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/configure-low-disk-space-alert-performance-logs is the recommended workaround.
